cmake/Platform/Linux/Packaging/snapcraft.yaml.in

```diff
@@ -0,0 +1,64 @@
+name: o3de
+version: '${LY_VERSION_STRING}'
+summary: O3DE Engine
+description: |
+  O3DE Engine
+confinement: classic
+base: core22
+
+parts:
+  test-o3de:
+    plugin: dump
+    source: ./O3DE
+    source-type: local
+    stage-packages:
+     - clang
+     - ninja-build
+     - libatomic1
+     - libfreetype6
+     - libpng16-16
+     - libgl1
+     - libxcb-icccm4
+     - libxcb-image0
+     - libxcb-keysyms1
+     - libxcb-randr0
+     - libxcb-render-util0
+     - libxcb-render0
+     - libxcb-shape0
+     - libxcb-xinerama0
+     - libxcb-xinput0
+     - libxcb-xkb1
+     - libxkbcommon-x11-0
+     - libxkbcommon0
+     - libpcre2-16-0
+     - libopengl0
+     - libice6
+     - libsm6
+     - libcurl4
+     - libfontconfig1
+    stage-snaps:
+     - cmake
+
+layout:
+  /etc/o3de/${LY_VERSION_STRING}/downloaded_packages:
```

Comment:
you probably want to replace all of the `${LY_VERSION_STRING}` with `${O3DE_INSTALL_VERSION}`, if you're looking for a unique version number, or `${O3DE_INSTALL_DISPLAY_VERSION}` if looking for the release version